CDP4WebServer.IIS/Web.config

```diff
@@ -0,0 +1,68 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <appSettings>
+    <!-- OWIN is used, so all calls should be handled by OWIN -->
+    <add key="owin:HandleAllRequests" value="true"/>
+    <add key="owin:appStartup" value="CDP4WebServer.IIS.Startup, CDP4WebServer.IIS" />
+  </appSettings>
+    <system.web>
+    <compilation debug="true" targetFramework="4.7.2" />
+    <!-- E-CSS-TM-10-25 defines wildcard URL's that contain an asterisk (*) -->
+    <httpRuntime maxUrlLength="4096" maxQueryStringLength="4096" requestPathInvalidCharacters="&lt;,&gt;,%,&amp;,:,\,?" />
```

Comment:
this may not work in all cases. Due to the nesting of requirement groups the length could be infinite. Can we make this a bit longer so we do not hit this limit too fast?

Reply:
Ok, I'll try to find the limits.

Reply:
Set both to 16384.